pages/index.tsx

```diff
@@ -1,39 +1,58 @@
-import { HStack, Stack, Text } from "@chakra-ui/react";
+import {
+  Box,
+  Button,
+  Divider,
+  HStack,
+  Stack,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
 import Layout from "@components/Layout";
 import pageTitle from "@lib/helpers/frontend/page-title";
 import { useUserQuery } from "@lib/hooks/user";
 import type { NextPage } from "next";
 import Head from "next/head";
 import Logo from "@components/Logo";
 import { SearchBar } from "@components/SearchBar";
+import Link from "next/link";
 
 const Home: NextPage = () => {
-  const { user } = useUserQuery();
-
   return (
     <>
       <Head>
         <title>{pageTitle()}</title>
         <meta name="description" content="Generated by create next app" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Layout hideHeaderSearch={true} marginY={"130px"}>
-        <Stack align={"center"}>
-          <HStack spacing={5} align={"center"}>
-            <Logo />
-            <Text fontSize="6xl" color="accent" fontFamily="title">
-              Book Bazar
-            </Text>
-          </HStack>
-          <Stack spacing={10} align={"center"}>
-            <SearchBar openOnFocus={true} autoFocus overlay={false} />
-          </Stack>
-        </Stack>
-        {user && (
-          <pre style={{ whiteSpace: "pre-wrap" }}>
-            {JSON.stringify(user, null, 2)}
-          </pre>
-        )}
+      <Layout minimalHeader>
+        <VStack mt={{ base: "20", md: "36" }} spacing="12">
+          <VStack spacing="4" width="100%">
+            <HStack
+              spacing={{ base: 3, md: 5 }}
+              fontSize={{ base: "4xl", md: "6xl" }}
+            >
+              <Logo />
+              <Text
+                color="accent"
+                fontFamily="title"
+                fontWeight="500"
+                whiteSpace="nowrap"
+              >
+                Book Bazar
+              </Text>
+            </HStack>
+            <Box margin="auto" maxWidth="40rem" width="100%">
+              <SearchBar openOnFocus={true} autoFocus overlay={true} />
+            </Box>
+          </VStack>
+          <Divider width="80%" maxWidth="sm" />
+
+          <Link href="/create-post" passHref>
+            <Button as="a" colorScheme="teal">
+              Sell your used textbooks!
```

Comment:
Only comment I have is for the button under the search bar. The header already has a "Sell book" button.

Should we have this too?

Reply:
Yeah I don't think it hurts and kind of highlights the two actions you can take on the site. Do you think it should be removed?

Reply:
Ya, I think it should be removed. The centre piece of the page is the search bar, and the button takes away from that.

Reply:
FYI, if I click on the header's "sell book" button, the search bar pops up in the header on the main page.